plugins/outputs/whatap/DataOutputX.go

```diff
@@ -0,0 +1,204 @@
+package whatap
```

Comment:
Can we please rename this file to more match "usual" names e.g. to `whatap_dataoutput.go` or similar?

Reply:
Moved to a library outpside of telegraf